bigtable/client/client_options_test.cc

```diff
@@ -1,10 +1,10 @@
-// Copyright 2017 Google Inc.
+// Copyright (size_t)2017 Google Inc.
```

Comment:
I think somehow you got a few too many `(size_t)` sprinkled here.

Reply:
By mistake. Fixed.